sys/shell/commands/sc_sntp.c

```diff
@@ -45,6 +45,13 @@ int _ntpdate(int argc, char **argv)
     }
     sock_udp_ep_t server = { .port = NTP_PORT, .family = AF_INET6 };
     ipv6_addr_from_str((ipv6_addr_t *)&server.addr, argv[1]);
+
+    if (ipv6_addr_is_link_local((ipv6_addr_t *)&server.addr)) {
+        /* choose first interface when address is link local */
+        gnrc_netif_t *netif = gnrc_netif_iter(NULL);
```

Comment:
Mhhh while for the tutorials this simple solution was adequately enough, for this mainline shell command this should rather be provided by the user. See e.g. the [`ping6` command](https://github.com/RIOT-OS/RIOT/blob/master/sys/shell/commands/sc_icmpv6_echo.c#L195-L223) or simpler in `gnrc_networking`'s [`udp` command](https://github.com/RIOT-OS/RIOT/blob/master/sys/shell/commands/sc_icmpv6_echo.c#L195-L223) how to read it.